cmd/ops/deprecation.go

```diff
@@ -26,6 +26,7 @@ func init() {
 	deprecationsCmd.Flags().StringArrayVar(&flags.schemaFiles, schemaFileFlagName, []string{}, "Server's schema as file or url. Can be repeated (required)")
 	deprecationsCmd.MarkFlagRequired(schemaFileFlagName) //nolint:errcheck // will err if flag doesn't exist
 	deprecationsCmd.Flags().StringArrayVar(&flags.ignore, ignoreFlagName, []string{}, "Files to ignore. Can be repeated")
+	deprecationsCmd.Flags().BoolVarP(&flags.verbose, verboseFlagName, "v", false, "Verbose mode. Will print debug messages")
```

Comment:
Wouldn't it make more sense to make it global? like `--output`. That way we can define it only once

Reply:
That's a very good point - it should absolutely be a global flag! Fixed in 5afd27f01d4beb3797f3ac734027073a2189c1ce.